worldedit-core/src/main/java/com/sk89q/worldedit/world/registry/ItemRegistry.java

```diff
@@ -19,19 +19,6 @@
 
 package com.sk89q.worldedit.world.registry;
 
-import com.sk89q.worldedit.blocks.BaseItem;
-
-import javax.annotation.Nullable;
-
 public interface ItemRegistry {
```

Comment:
Is there even a point to this interface now? It's totally empty.

Reply:
Just fixed it up, it's now used like it should be